Serenity-Website/src/Sign-up/SignUp.css

```diff
@@ -124,330 +121,3 @@ body{
     color: deeppink;
 }
 
-/* 
```

Comment:
Please remove all commented-out code! A PR request is a request to merge code into master. Master code should never contain commented-out code (unless intentionally left in for explanatory purposes).

Reply:
✅